inference-engine/src/transformations/src/transformations/common_optimizations/convert_nms_gather_path_to_unsigned.cpp

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/convert_nms_gather_path_to_unsigned.hpp"
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/opsets/opset3.hpp>
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/op/util/broadcast_base.hpp>
+#include <ngraph/op/util/gather_base.hpp>
+#include <ngraph/rt_info.hpp>
+#include <memory>
+#include "itt.hpp"
+#include "ngraph/node.hpp"
+
+using namespace ngraph;
+using namespace std;
+
```

Comment:
nit: please add some description for each pass in this file: InitNMSPath, PropagateNMSPath, UpdateConvertGather